support/mender-inventory-hostinfo

```diff
@@ -17,5 +17,5 @@ cat /proc/meminfo | awk '
 /MemTotal/ {printf("mem_total_kB=%d\n", $2)}
 '
 
-echo "hostname=$(cat /etc/hostname)"
+echo "hostname=$(hostname)"
```

Comment:
maybe lets check if we have `hostname` command, if we do, then use it, if not, then fall back to `/etc/hostname`, and if that fails, we can even grep in the /etc/hosts for the first word in a first uncommented line for 127.0.0.1, and if that fails we fall back to `localhost` ? just a nit pick.

Reply:
sorry for late answer, I only got ot it now. perhaps:
```bash
hostname="localhost"
hostname >/dev/null 2>&1 && hostname="$(hostname)"
[ "$hostname" == "" ] && [ -f /etc/hostname ] && hostname="cat /etc/hostname 2>/dev/null"
hostname=${hostname:-"localhost"}
```